### PR TITLE
SPLAT-1626: add AWS platform external workflow to troubleshoot

### PR DESCRIPTION
**Pull request by rvanderp3**
### Description

Introducing a new workflow to clusterbot to add in the debug of [SPLAT-1628](https://issues.redhat.com//browse/SPLAT-1628).

### CODE REVIEW

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -625,6 +625,8 @@ workflows:
     platform: vsphere
   openshift-psap-e2e-aws-gpu:
     platform: aws
+  openshift-e2e-external-aws:
+    platform: aws
```
Comment (mtulio) [inline core-services/ci-chat-bot/workflows-config.yaml:629]: should we change this for `external` or something like `external-aws`?

Comment (rvanderp3) [reply core-services/ci-chat-bot/workflows-config.yaml:629]: i think this just ensures that it lands on the build cluster

**Pull request merged**